resources/add_to_list.rb

```diff
@@ -1,4 +1,4 @@
-property :backup, [true, false], default: false
+property :backup, [true, false, Integer], default: false
```

Comment:
You're going to have to talk me through now a boolean needs an Integer value

Reply:
Backup gets passed to the file resource.  File only accepts false or Integer.  I didn't read the file resource doc closely enough and thought it was a boolean.  If I hadn't advertised this as allowing true I'd probably go back to matching the file resource and only allow false or Integer.  The design of file is nuts though.  Allowing an integer and false, but not true is just wrong.

Line historically has not taken backups.

Reply:
I deleted the checks that the backup files were created.  We don't need to check that the file resource creates backups.  The tests should pass now.